example/src/AdInspector.tsx

```diff
@@ -0,0 +1,45 @@
+import {Test, TestType, TestResult} from 'jet';
+import React from 'react';
+import {View, Button, StyleSheet} from 'react-native';
+import MobileAds from 'react-native-google-mobile-ads';
+
+export class AdInspectorTest implements Test {
+  getPath(): string {
+    return 'AdInspectorTest';
+  }
```

Comment:
I hate to argue for "worse" code, but could you put this directly in the App.tsx? The reason is that the example app is built by a script, and it has to copy relevant files out while it rebuilds from react-native init, so it's easier to keep the example just as one file for copy out / copy in

That's why the file is huge and everything is in there all in one spot :-)

Reply:
Okay, I understand.